src/components/Questionnaire/QuestionnaireForm.tsx

```diff
@@ -77,10 +78,19 @@ export function QuestionnaireForm({
     prefetch: !!questionnaireSlug && !FIXED_QUESTIONNAIRES[questionnaireSlug],
   });
 
-  const { mutate: submitBatch, isProcessing } = useMutation(
-    routes.batchRequest,
-    { silent: true },
-  );
+  const { mutate: submitBatch, isPending: isProcessing } = useMutation({
+    mutationFn: mutate(routes.batchRequest),
```

Comment:
we had silenced errors previously because we are writing a custom handler for handling the error. Did you miss this out?

```suggestion
    mutationFn: mutate(routes.batchRequest, { silent: true }),
```